vote/blockchain.py

```diff
@@ -0,0 +1,107 @@
+from hashlib import sha256 
```

Comment:
Please take this from the project. It is a separate project on it own

Reply:
tried to combine the flask and the django project at some point , and reverted , will be addressed in the next pr